posixlib/src/main/resources/scala-native/time.c

```diff
@@ -91,6 +106,14 @@ size_t scalanative_strftime(char *buf, size_t maxsize, const char *format,
     return strftime(buf, maxsize, format, &tm);
 }
 
+char *scalanative_strptime(const char *s, const char *format,
```

Comment:
Is this function used anywhere? I cannot see `@name` annotation using it in `posix/time.scala`

Reply:
@WojciechMazur  you are right, that is a BUG!  Thank you for spotting it.
It was correct in the original PR #1667 but I missed carrying it forward.
Bad dog, no cookie...

I added a block of comment to describe why some lines have `!name` and
some do not.  It is an understanding that has evolved over time. The lack
of any clear statement or decision rule has lead to many a bug, even
by those who at one time knew the lore.

The additional whitespace makes it easier for, at least me, to find
the name annotations and remember/figure-out the grouping.
I can do that in a separate PR if that is advised.

Reply:
In working on this, I wondered why the original PR #1667 used a .c file to do the 
localtime_r and strftime calls, when obviously it could have been done in scala.

The reason, never explained, is to avoid having to adjust a C structure to a 
ScalaNative structure and then immediately turn around and adjust the SN
structure back to a C structure. 

I think going with the currently proposed Scala code is the way to go, but
doing the archeology helps me understand what appeared to be an
obvious blunder.  It also helps refresh the idea that  structures
in Scala Native can be a performance hit.